docs/extend/modular-resources/key-concepts.md

```diff
@@ -0,0 +1,91 @@
+
+### APIs
+
+{{% alert title="Modules vs. modular resources" color="tip" %}}
+
+A configured *module* can make one or more *modular resources* available for configuration.
+
+{{% /alert %}}
+
+Every Viam {{< glossary_tooltip term_id="resource" text="resource" >}} subtype exposes an [application programming interface (API)](https://en.wikipedia.org/wiki/API).
+This can be understood as a description of how you can interact with that resource.
+Each API is described through [protocol buffers](https://developers.google.com/protocol-buffers).
+Viam SDKs [expose these APIs](/internals/robot-to-robot-comms/).
+
+#### Namespace
+
+Each Viam resource's API is uniquely namespaced as a colon-delimited-triplet in the form of `namespace:type:subtype`.
+
+For example:
+
+- The API of built-in component [camera](/components/camera/) is `rdk:component:camera`, which exposes methods such as `GetImage()`.
+- The API of built-in service [vision](/services/vision/) is `rdk:service:vision`, which exposes methods such as `GetDetectionsFromCamera()`.
+
+{{% alert title="Tip" color="tip" %}}
+You can see built-in Viam resource APIs in the [Viam GitHub](https://github.com/viamrobotics/api).
+{{% /alert %}}
+
+### Models
+
+A *model* describes a specific implementation of a resource that implements (speaks) its API.
+Models allow you to control different instances of resource {{< glossary_tooltip term_id="api-namespace-triplet" text="subtypes" >}} with a consistent interface.
+
+For example:
+
+Some DC motors use just [GPIO](/components/board/), while other DC motors use serial protocols like [SPI bus](/components/board/#spis).
+Regardless, you can power any motor model that implements the *rdk:component:motor* API with the `SetPower()` method.
+
+#### Namespace
+
+Models are also uniquely namespaced as colon-delimited-triplets in the form of `namespace:family:name`.
+
+For example:
+
+- The `rdk:builtin:gpio` model of the `rdk:component:motor` API provides RDK support for [GPIO-controlled DC motors](/components/motor/gpio/).
+- The `rdk:builtin:DMC4000` model of the same `rdk:component:motor` API provides RDK support for the [DMC4000](/components/motor/dmc4000/) motor.
+
+### Management
+
+The Robot Development Kit (RDK) `viam-server` provides automatically manages modular resources to function like built-in resources:
+
+#### Dependency Management
+
+Modular resources may depend on other built-in resources or other modular resources, and vice versa.
+The Viam RDK handles dependency management.
+
+#### Start-up
+
+The RDK ensures that any configured modules are loaded automatically on start-up, and that configured modular resource instances are started alongside configured built-in resource instances.
+
+#### Reconfiguration
```

Comment:
```suggestion
### Reconfiguration
```

Reply:
Wait, why do you want to change all of these headings to be visible in the sidebar? Is this necessary?

Reply:
There is no H2 above them all, so why make them so small? 

Additionally, there's less of a different between H3 and H4 than there is between H2 and H3 (at least to my eye), so it is easier to parse when it's that way, and here, why not??